cql3/statements/strongly_consistent_select_statement.hh

```diff
@@ -10,17 +10,22 @@
 
 #pragma once
 
+#include "cql3/expr/expression.hh"
 #include "cql3/statements/select_statement.hh"
 #include "service/broadcast_tables/experimental/lang.hh"
 
 namespace cql3 {
 
 namespace statements {
 
+struct prepared_select_query {
```

Comment:
Same naming/namespacing issue as for updates.

Reply:
Done.